src/db.c

```diff
@@ -242,7 +242,8 @@ int getKeySlot(sds key) {
      */
     if (server.current_client && server.current_client->slot >= 0 && server.current_client->flag.executing_command &&
         !server.current_client->flag.primary) {
-        debugServerAssertWithInfo(server.current_client, NULL, (int) keyHashSlot(key, (int)sdslen(key)) == server.current_client->slot);
+        debugServerAssertWithInfo(server.current_client, NULL,
+                                  (int)keyHashSlot(key, (int)sdslen(key)) == server.current_client->slot);
```

Comment:
The way clang-format decides where to add linebreaks is annoying sometimes.

Here, and in config.c, we just want the lines to be longer since what's on the far right is usually not important to see.

Other times, there may be a logical way to arrange arguments, like grouping some arguments together:

```C
    printf("First: %d, second: %d, third: %d\n",
           first, second, third);
```

Clang-format reformats this and puts as much as possible on one line up to our 120 limit.

In libvalkey, we decided to go with `ColumnLimit: 0`. This disables the reflowing and clang-format get's much less annoying.

Reply:
I Agree, this is the one thing I don't like dealing with. I saw Mikal talking about it in the developer slack.